stock_available/product.py

```diff
@@ -38,7 +38,7 @@ def _immediately_usable_qty(self):
             ('usage', '=', 'internal')])
         sublocation_ids = []
         for location in internal_locations:
-            sublocation_ids.append(self.env['stock.location'].search(
+            sublocation_ids.extend(self.env['stock.location'].search(
```

Comment:
2015-08-05 3:59 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> sublocations = self.env['stock.location'] sublocations +=
> self.env['stock.location'].search(..) ... ('location_id', 'in',
> sublocations.ids),


+1 for this comment.


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>